blazor/sankey/legend.md

```diff
@@ -0,0 +1,263 @@
+            new SankeyDataLink() { SourceId = "Natural Gas", TargetId = "Electricity", Value = 130 },
+            new SankeyDataLink() { SourceId = "Natural Gas", TargetId = "Heat", Value = 80 },
+            new SankeyDataLink() { SourceId = "Oil", TargetId = "Fuel", Value = 250 },
+
+            // Energy Carriers to Sectors
+            new SankeyDataLink() { SourceId = "Electricity", TargetId = "Residential", Value = 170 },
+            new SankeyDataLink() { SourceId = "Electricity", TargetId = "Commercial", Value = 160 },
+            new SankeyDataLink() { SourceId = "Electricity", TargetId = "Industrial", Value = 210 },
+            new SankeyDataLink() { SourceId = "Heat", TargetId = "Residential", Value = 40 },
+            new SankeyDataLink() { SourceId = "Heat", TargetId = "Commercial", Value = 20 },
+            new SankeyDataLink() { SourceId = "Heat", TargetId = "Industrial", Value = 20 },
+            new SankeyDataLink() { SourceId = "Fuel", TargetId = "Transportation", Value = 200 },
+            new SankeyDataLink() { SourceId = "Fuel", TargetId = "Industrial", Value = 50 },
+
+            // Sectors to End Use and Losses
+            new SankeyDataLink() { SourceId = "Residential", TargetId = "Energy Services", Value = 180 },
+            new SankeyDataLink() { SourceId = "Commercial", TargetId = "Energy Services", Value = 150 },
+            new SankeyDataLink() { SourceId = "Industrial", TargetId = "Energy Services", Value = 230 },
+            new SankeyDataLink() { SourceId = "Transportation", TargetId = "Energy Services", Value = 150 },
+            new SankeyDataLink() { SourceId = "Residential", TargetId = "Losses", Value = 30 },
+            new SankeyDataLink() { SourceId = "Commercial", TargetId = "Losses", Value = 30 },
+            new SankeyDataLink() { SourceId = "Industrial", TargetId = "Losses", Value = 50 },
+            new SankeyDataLink() { SourceId = "Transportation", TargetId = "Losses", Value = 50 }
+        };
+        base.OnInitialized();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+
+![Blazor Sankey Basic Legend](images/legends/sankey-legends.png)
+
+## Legend Customization and Configuration
+
+The Sankey diagram component offers a robust set of features for legend customization and configuration, allowing users to create visually appealing and informative representations of their data. This comprehensive toolset encompasses a wide range of options, from basic visibility controls to advanced styling capabilities.
+
+Appearance customization extends beyond basic properties, allowing for detailed adjustments to the legend's visual elements. Users can modify the dimensions of legend shapes, customize the padding between shapes and text, and even invert or reverse the order of legend items. The legend can be further enhanced with a title, adding context to the displayed information.
+
+###  Key Properties
```

Comment:
Chekc this heading needed, we need the properties can we explain along the description.